### PR TITLE
fix: use container spacing for resource page

### PR DESCRIPTION
**Pull request by njhuey**
Alter styles in ApiResources.tsx to use React Bootstrap Container spacing for mobile viewing.  

Addresses issue #69

### CODE REVIEW

File: apps/site/src/views/Resources/sections/ApiResources/ApiResources.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ function ApiResources() {
 		<Container>
 			<Row className={styles.row}>
 				{/* Card Component */}
-				<Col className={styles.column}>
+				<Col className={styles.column} xl={12}>
```
Comment (taesungh) [inline apps/site/src/views/Resources/sections/ApiResources/ApiResources.tsx:15]: Thought: I feel like the `xl` here wouldn't change the layout?

Comment (taesungh) [inline apps/site/src/views/Resources/sections/ApiResources/ApiResources.tsx:15]: Actually, noticing the API Resources card should not be in the row together with the resource cards

**Review by taesungh — COMMENTED** [review]
Think a similar modification would also help with the intro card in `BackendResources`.

**Review by samderanova — DISMISSED** [review]
Thanks for the quick fix! The text overflows the borders of the backend frameworks, but this can be fixed in a different issue.

File: apps/site/src/views/Resources/sections/ApiResources/ApiResources.tsx [hunk 2]
```diff
@@ -10,9 +10,9 @@ import ApiResourcesList from "./config";
 function ApiResources() {
 	return (
 		<Container>
-			<Row className={styles.row}>
+			<Row>
 				{/* Card Component */}
-				<Col className={styles.column} xl={12}>
+				<Col className={styles.column}>
```
Comment (taesungh) [inline apps/site/src/views/Resources/sections/ApiResources/ApiResources.tsx:15]: Thought: at this point with only one row element, do we even need the column and row anymore?

File: apps/site/src/views/Resources/sections/BackendResources/BackendResources.tsx [hunk 3]
```diff
@@ -11,18 +11,16 @@ import BackendResourcesList from "./config";
...
-						<h2 className={styles.title}>Backend Framework Resources</h2>
-						<p className={styles.text}>
-							Backend Frameworks are a variety of middleware services used to
-							connect to other API and database vendors to fit your
-							project&apos;s needs.
-						</p>
-					</div>
-				</Col>
+			{/* Card Component */}
+			<div className={styles.card}>
```
Comment (taesungh) [inline apps/site/src/views/Resources/sections/BackendResources/BackendResources.tsx:15]: Note: still need to remove the `width: 80vw` style.

**Review by samderanova — APPROVED** [review]
Good work. Let's merge this so development on the resources page can continue.

**Pull request merged**